src/languages/ruby.md

```diff
@@ -4,8 +4,8 @@ Platform.sh supports deploying any Ruby application. Your application can use
 any Ruby application server such as Unicorn or Puma and deploying a Rails or
 a Sinatra app is very straight forward.
 
-### Supported versions
-#### Ruby MRI
+## Supported versions
```

Comment:
We could say it's MRI without using an extra header for it.  Say, in the paragraph above.